pkg/minikube/console/style_enum.go

```diff
@@ -0,0 +1,147 @@
+/*
+Copyright 2019 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package console
+
+type StyleEnum int
+
+const (
+	Happy StyleEnum = iota
+	SuccessType
+	FailureType
+	Conflict
+	FatalType
+	Notice
+	Ready
+	Running
+	Provisioning
+	Restarting
+	Reconfiguring
+	Stopping
+	Stopped
+	WarningType
+	Waiting
+	WaitingPods
+	Usage
+	Launch
+	Sad
+	ThumbsUp
+	Option
+	Command
+	LogEntry
+	Crushed
+	URL
+	Documentation
+	Issues
+	Issue
+	Check
+	IsoDownload
+	FileDownload
+	Caching
+	StartingVM
+	StartingNone
+	Resetting
+	DeletingHost
+	Copying
+	Connectivity
+	Internet
+	Mounting
+	Celebrate
+	ContainerRuntime
+	Docker
+	Crio
+	Containerd
+	Permissions
+	Enabling
+	Shutdown
+	Pulling
+	Verifying
+	VerifyingNoLine
+	Kubectl
+	Meh
+	Embarrassed
+	Tip
+	Unmount
+	MountOptions
+	Fileserver
+)
+
+func (style StyleEnum) String() string {
```

Comment:
Any chance we could avoid implementing the String() function? I'm not clear who uses it.

Reply:
Done.